src/devices/Pcd8544/ScreenTemperature.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display.Pcd8544Enums
+{
+    /// <summary>
+    /// Temperature set for the PCD8544
+    /// </summary>
+    public enum ScreenTemperature
+    {
+        /// <summary>Temperature Coefficient 0</summary>
+        Coefficient0 = 0b0000_0100,
+
+        /// <summary>Temperature Coefficient 1</summary>
+        Coefficient1 = 0b0000_0101,
+
+        /// <summary>Temperature Coefficient 2</summary>
+        Coefficient2 = 0b0000_0110,
+
+        /// <summary>Temperature Coefficient 3</summary>
+        Coefficient3 = 0b0000_0111,
```

Comment:
If there is more info in the docs what those coefficients mean consider adding those info in the docs. Currently it's hard to tell what this means

Reply:
I would love to have more info! The doc just give them and recommend to use the default I'm using and mentioned in the readme. The sample goes thru all modes and covers everything. So everyone can understand the impact of each element.